code/datums/gamemodes/_game_mode.dm

```diff
@@ -888,6 +888,13 @@ GLOBAL_LIST_INIT(bioscan_locations, list(
 
 ///Add gamemode related items to statpanel
 /datum/game_mode/proc/get_status_tab_items(datum/dcs, mob/source, list/items)
+	var/patrol_end_countdown = game_end_countdown()
+	if(patrol_end_countdown)
+		items += "Round End timer: [patrol_end_countdown]"
+	var/patrol_wave_countdown = wave_countdown()
+	if(patrol_wave_countdown)
+		items += "Respawn wave timer: [patrol_wave_countdown]"
```

Comment:
this should be in the combat patrol child proc.

Reply:
I guess its fine for now since the wording is generic.

But the countdowns are procs not vars. While the procs exist in the base gamemode, they don't do anything (but could be used in other gamemodes in the future of course) so this part of the stat should be where its actually used. i.e. combat patrol gamemode. But yeah, fine for now.